projects/standard-rulesets/src/breaking-changes/preventArrayItemLengthChanges.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  RequestRule,
+  ResponseBodyRule,
+  RuleError,
+} from '@useoptic/rulesets-base';
+import {
+  arrayMaxItemsShouldNotIncrease,
+  arrayMinItemsShouldNotDecrease,
+  numericMaximumRangeShouldNotExpand,
+  numericMinimumRangeShouldNotExpand,
+} from './helpers/rangeChange';
+
+export const preventRequestArrayLengthChanges = new RequestRule({
+  name: 'prevent request array length changes',
+  rule: (requestAssertions) => {
+    requestAssertions.body.changed((before, after) => {
+      arrayMaxItemsShouldNotIncrease(
+        before.value.flatSchema,
+        after.value.flatSchema
+      );
+    });
+
+    requestAssertions.body.changed((before, after) => {
+      arrayMinItemsShouldNotDecrease(
+        before.value.flatSchema,
+        after.value.flatSchema
+      );
+    });
```

Comment:
I don't understand this - why is expanding the range a breaking change? Shouldn't it be the opposite?

Reply:
If I'm expecting at least 3 items in the array -- let's say to make a "Top Destinations" view, I don't care if that list can have 5, 10, 20 items because I'm pulling the top 3. 

If the minimum goes down that breaks me...? maybe. 

it's a weird use case. I don't think most people use these keywords, but those who do are probably building expectations around the min / max numbers into their code.